simpledotorg/simple-server
# 3327 who report

**Story card:** [ch3327](https://app.clubhouse.io/simpledotorg/epic/3327)

## Because

The WHO wants reports in a specific format. 

## This addresses

This produces a csv with data for a district and its facilities. 

## Test instructions

- Go to the reports tab and navigate to a district
- click the download dropdown
- there should be a new option "Monthly district data"
- click that and accept the download
- expect it to be populated with data for the district and its facilities
